src/main/java/fr/xephi/authme/datasource/MySQL.java

```diff
@@ -566,6 +566,44 @@ public void setUnlogged(String user) {
     }
```

Comment:
Initialization of `hasSession` if missing should be added to `checkTablesAndColumns()`

Reply:
Yeah, again, i shouldn't code when i'm on a crowded train xD